TopAnalysis/interface/FCNCTriLeptonCppWorker.h

```diff
@@ -22,7 +22,7 @@ class FCNCTriLeptonCppWorker {
   ~FCNCTriLeptonCppWorker() = default;
 
   void setMuons(TRAF pt, TRAF eta, TRAF phi, TRAF mass, TRAI charge,
-                TRAF relIso, TRAB isTight, TRAB isGlobal, TRAB isPFcand, TRAB isTracker);
+                TRAF relIso, TRAB isLoose, TRAB isTight, TRAB isGlobal, TRAB isPFcand, TRAB isTracker);
   void setElectrons(TRAF pt, TRAF eta, TRAF phi, TRAF mass, TRAI charge,
                     TRAF relIso, TRAI id, TRAF dEtaSC, TRAF eCorr);
```

Comment:
need lots of modification in the setElectron to invert isolation. electrion id code is just for the isolated electrons.